src/components/WorkOrder/__snapshots__/WorkOrderHeader.test.js.snap

```diff
@@ -0,0 +1,493 @@
+        </div>
+        <div
+          class="lbh-body-xs govuk-!-margin-bottom-2"
+        >
+          <span>
+            Caller: Jill Smith
+          </span>
+          <br />
+          <span>
+            07700 900999
+          </span>
+        </div>
+      </div>
+    </div>
+    <div
+      class="govuk-grid-column-one-third"
+    >
+      <div
+        class="appointment-details"
+      >
+        <span
+          class="govuk-!-font-size-14"
+        >
+          Appointment details
+        </span>
+        <br />
+        <div
+          class="lbh-body-s"
+        >
+          <span
+            class="lbh-!-font-weight-bold"
+          >
+            Not applicable
+          </span>
+        </div>
+      </div>
+      <div
+        class="lbh-body-xs"
+      >
+        <span>
+          Assigned to: Alphatrack (S) Systems Lt
+        </span>
+      </div>
+      <div
+        class="lbh-body-xs"
+      >
+        <span>
+          <strong>
+            : 22 Jan 2021, 18:15
+          </strong>
+        </span>
+      </div>
+    </div>
+  </div>
+</DocumentFragment>
+`;
+
+exports[`WorkOrderHeader component when workOrder status is Work Complete or No Access should show complition reason: Completed, date and time 1`] = `
+<DocumentFragment>
+  <div
+    class="lbh-body-s govuk-grid-row"
+  >
+    <div
+      class="govuk-grid-column-one-third"
+    >
+      <div
+        class="property-details-main-section"
+      >
+        <span
+          class="govuk-!-font-size-14"
+        >
+          Dwelling
+        </span>
+        <br />
+        <a
+          class="lbh-link"
+          href="/properties/undefined"
+        >
+          <div
+            class="lbh-!-font-weight-bold"
+          >
+            16 Pitcairn House
+            <br />
+            St Thomass Square
+            <br />
+            <span
+              class="govuk-!-font-size-14"
+            >
+              E9 6PT
+            </span>
+          </div>
+        </a>
+      </div>
+      <ul
+        class="lbh-list hackney-property-alerts"
+      >
+        <li
+          class="bg-dark-green"
+        >
+          Tenure: Secure
+        </li>
+        <li
+          class="bg-orange"
+        >
+          Address Alert: Property Under Disrepair (
+          <strong>
+            DIS
+          </strong>
+          )
+        </li>
+        <li
+          class="bg-orange"
+        >
+          Contact Alert: Property Under Disrepair (
+          <strong>
+            DIS
+          </strong>
+          )
+        </li>
+      </ul>
+    </div>
+    <div
+      class="govuk-grid-column-one-third"
+    >
+      <div
+        class="work-order-info"
+      >
+        <div
+          class="lbh-body-s govuk-!-margin-bottom-2"
+        >
+          <span
+            class="circle-tag circle-tag--status-work-complete"
+          />
+          <span
+            class="lbh-!-font-weight-bold"
+          >
+            Status: Work Complete
```

Comment:
I think this will need to be `Status: Complete`. This might be resolved by the addition of the `return` to the helper function mentioned above.

Reply:
On revisiting this I realised I wasn't looking in the right place in the snapshot for the output of that function. I can see that we show the expected text [here](https://github.com/LBHackney-IT/repairs-hub-frontend/pull/286/files#diff-6de5dff3be570d9c44b4d49d8ba242529217710cbb3cc6f88324e22866f28f11R322).